### PR TITLE
gRPC-core cleanup

### PR DESCRIPTION
**Pull request by singhravidutt**
Cleaning up the old grpc-core integration in favour of java-storage client integration.

### CODE REVIEW

File: gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java [hunk 1]
```diff
@@ -428,40 +410,22 @@ public WritableByteChannel create(StorageResourceId resourceId, CreateObjectOpti
...
-            @Override
-            public Storage.Objects.Insert createRequest(InputStreamContent inputStream)
-                throws IOException {
-              return initializeRequest(
-                  super.createRequest(inputStream), resourceId.getBucketName());
-            }
-          };
-    }
+
+    channel =
```
Comment (arunkumarchacko) [inline gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java:414]: Please make assignment part of the declaration

File: gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java [hunk 2]
```diff
@@ -341,13 +335,6 @@ public Boolean load(String bucketName) {
         MetricsSink.CLOUD_MONITORING == options.getMetricsSink()
             ? CloudMonitoringMetricsRecorder.create(options.getProjectId(), finalCredentials)
             : new NoOpMetricsRecorder();
-
-    // Create the gRPC stub if necessary;
-    if (options.isGrpcEnabled()) {
```
Comment (arunkumarchacko) [inline gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java:346]: Let use remove the java stroage flag and use gRPC flag to use Veneer client? grpc.enable flag will not have any effect, I guess?

**Review by arunkumarchacko — APPROVED** [review]

File: gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java (hunk 1 above)
Comment (singhravidutt) [reply gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java]: updated

File: gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java (hunk 2 above)
Comment (singhravidutt) [reply gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java:346]: I am keeping track of it. As other flag is already shared with customer in preview phase will not change it immediately. Will be updating it in follow up PR.

**Pull request merged**